packages/atlas/src/components/_overlays/Popover/Popover.tsx

```diff
@@ -90,6 +92,9 @@ const _Popover: React.ForwardRefRenderFunction<PopoverImperativeHandle | undefin
           {children}
         </ContentContainer>
       )}
+      popperOptions={{
+        modifiers: [{ name: 'flip', enabled: flipEnabled }],
```

Comment:
I'm wondering, do we actually want this flip behavior enabled in the entire app? I think we should disable this and not worry about passing the `flipEnabled` prop.

Reply:
What exactly are we discussing here?

Reply:
If you look at Adam's video you'll notice that the popover changes his position when there is not much space for this context menu in the viewport. And I'm guessing this is the default behavior in Popover

https://user-images.githubusercontent.com/35307309/166924214-d2fa225e-3f0c-4f72-8991-7cea15ecb2fb.mp4

My proposition was to just turn off this behavior, and don't add a prop that could control this as Rafał did.

Reply:
@kdembler I'm affraid this won't be possible in the case above because of z-indexes.

Reply:
What do you mean? Can't we change those z-indexes?